dev-libs/libev/libev-4.31.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="High-performance event loop/event model with lots of features"
+DESCRIPTION="Libev is modelled (very losely) after libevent and the Event perl \
+module, but is faster, scales better and is more correct, and also more  \
+featureful. And also smaller."
+HOMEPAGE="http://software.schmorp.de/pkg/libev"
+COPYRIGHT="2007-2013 Marc Alexander Lehmann"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://dist.schmorp.de/libev/libev-$portVersion.tar.gz"
+CHECKSUM_SHA256="ed855d2b52118e32c0c1a6a32bd18c97f9e6711ca511f5ee12de3b9eccc66e5a"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libev$secondaryArchSuffix = $portVersion
+	lib:libev$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libev${secondaryArchSuffix}_devel = $portVersion
+	devel:libev$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libev$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -f $libDir/libev.la
+
+	prepareInstalledDevelLib libev
+
+	packageEntries devel \
+		$developDir
```

Comment:
Move manpage too, check for an example: https://github.com/haikuports/haikuports/blob/aa7c82bc3cd0185f2a493228ac2ed69de46c86e2/x11-libs/libx11/libx11-1.6.9.recipe#L155